Tests/backend-specific-tests/Teradata/RunnerTeradataTest.php

```diff
@@ -44,7 +44,7 @@ private function clearConfigs()
                     ->setConfigurationId($configuration['id'])
             );
             foreach ($workspaces as $workspace) {
-                $workspacesApi->deleteWorkspace($workspace['id'], ['async' => true]);
```

Comment:
Dobry ze nekdo mel aspon snahu :smile:

Reply:
Aha, ne to bylo dřív taky správně https://github.com/keboola/storage-api-php-client/blob/403fb6ccf769cf63cb52492cff781c8288d773f5/src/Keboola/StorageApi/Workspaces.php#L54 ten async option tam byl historicky, ale teď se to nově přidalo jako nový argument, tak jsem to přesunul ať je to všude stejně s createWorkspace.